### PR TITLE
Angular Web Component

### PR DESCRIPTION
**Pull request by ZhmenZH**

### CODE REVIEW

**Review by fkolar — DISMISSED** [review]

**Comment by fkolar** [comment]
@valorkin can we merge, not sure if you have something in progress ..

**Comment by valorkin** [comment]
@fkolar I am working on builder, I will rebase later on

File: packages/cflp-app/lib/cart.consumer.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { rpcProvider } from './rpc.connector';
+import { EventSubject } from './event.subject';
+
+const ChangeCartAction = 'changeCartState';
+const GetDefaultCartAction = 'defaultCartState';
```
Comment (valorkin) [inline packages/cflp-app/lib/cart.consumer.ts]: change `default` to `current` and in Theme too, sorry, I forgot to change it

File: packages/content-item-app/lib/cart.provider.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { rpcProvider } from './rpc.connector';
+
+const ChangeCartAction = 'changeCartState';
+const GetDefaultCartAction = 'defaultCartState';
```
Comment (valorkin) [inline packages/content-item-app/lib/cart.provider.ts]: here too `default` -> `current` please

File: packages/content-item-app/src/app/item-page/item-page.component.ts [hunk 3]
```diff
@@ -15,21 +15,19 @@ import {
...
   }
 
-  onAddToCard($event: MouseEvent) {
-    const m = new MapMessage('app:event');
-    m.set('type', 'add-to-card');
-    m.set('payload', {xx: 'aa'});
-
-    this.messaging.publish('app:event', m);
+  onAddToCart($event: MouseEvent): void {
+    console.log('onAddToCart event', );
```
Comment (valorkin) [inline packages/content-item-app/src/app/item-page/item-page.component.ts]: please remove if not intended

**Review by valorkin — CHANGES_REQUESTED** [review]
couple of minor changes please
except this looks good to me

**Review by valorkin — APPROVED** [review]
looks good enough to me
@fkolar ready to merge

**Review by fkolar — APPROVED** [review]

**Pull request merged**